client/src/components/History/History.module.scss

```
@@ -16,10 +16,11 @@
   flex-grow: 1;
 }
 
+/* WP-50: Font-size below was overwriting previous font-size establishments. This should now be rectified. */
```

```suggestion
/* WP-50: Font-size below was overwriting previous font-size establishments. This should now be rectified. */
```
These comments are super helpful, but I think the best place for them is in the commit message. I always think of the code change as the "how" and the commit message as the "why" if that makes sense.

---

Gotcha! I can do that.